chain/chain/src/flat_storage_creator.rs

```diff
@@ -2,21 +2,14 @@ use crate::{ChainStore, ChainStoreAccess, RuntimeAdapter};
 use crossbeam_channel::{unbounded, Receiver, Sender};
 use near_chain_primitives::Error;
 use near_primitives::types::{BlockHeight, ShardId};
-use near_store::flat_state::FlatStorageStateStatus;
+use near_store::flat_state::{FlatStorageStateStatus, NUM_PARTS_IN_ONE_STEP};
 use std::sync::{Arc, Mutex};
 use tracing::info;
 
-/// Number of parts to which we divide shard state for parallel traversal.
-// TODO: consider changing it for different shards, ensure that shard memory usage / `NUM_PARTS` < X MiB.
-#[allow(unused)]
-const NUM_PARTS: u64 = 4_000;
-
-/// Number of traversed parts during a single step of fetching state.
-#[allow(unused)]
-const PART_STEP: u64 = 50;
-
-/// Creates flat storage and tracks creation status for the given shard.
+/// If we launched a node with enabled flat storage but it doesn't have flat storage data on disk, we have to create it.
+/// This struct is responsible for this process for the given shard.
 pub struct FlatStorageShardCreator {
+    /// Status of flat storage creation process.
     pub status: FlatStorageStateStatus,
```

Comment:
```suggestion
    pub status: FlatStorageStateStatus,
```

The comment mostly repeates the name of the type, it doesn't add info